projects/media-viewer/src/lib/annotations/comment-set/comment-set-render.service.ts

```diff
@@ -6,7 +6,7 @@ export class CommentSetRenderService {
 
   redrawComponents(commentComponents: CommentComponent[], height: number, rotate: number, zoom: number) {
       let prevComment: CommentComponent;
-      this.sortComponents(commentComponents, height, rotate).forEach((comment: CommentComponent) => {
+      this.sortComponents(commentComponents, height / zoom, rotate).forEach((comment: CommentComponent) => {
```

Comment:
can we pass the zoom value and use it in the calculation, because it might be confusing passing it like this